src/subspace_store.c

```diff
@@ -116,6 +116,7 @@ ts_subspace_store_add(SubspaceStore *store, const Hypercube *hc, void *object,
 			 * create one now. (There will always be one for time)
 			 */
 			Assert(last != NULL);
+			/* NOLINTNEXTLINE(clang-analyzer-core.NullDereference) */
```

Comment:
Not sure why you need this: there is an assert on the line before.

Reply:
See my other comment on this.

Reply:
Assertions should not be enabled in release builds. See my other comment on this.